locals.tf

```diff
@@ -1,2 +1,4 @@
 locals {
+  groups_map      = { for g in data.google_cloud_identity_groups.groups.groups : g.group_key[0].id => g.name }
+  search_in_a_map = lookup(local.groups_map, var.group_email)
```

Comment:
what will happen when email is not in a `groups_map`?

Reply:
│ Error: Invalid function argument
│ 
│   on locals.tf line 3, in locals:
│    3:   search_in_a_map = lookup(local.groups_map, var.group_email)
│     ├────────────────
│     │ while calling lookup(inputMap, key, default...)
│     │ local.groups_map is object with 68 attributes
│ 
│ Invalid value for "inputMap" parameter: the given object has no attribute "d@g.pl".

Reply:
maybe we can handle that gracefully?

Reply:
yes, probably we can :D